packages/design-tokens/README.md

```diff
@@ -48,17 +46,19 @@ The package exports a `themePropType` which can be used to check the `theme` pro
 
 ```js
 import PropTypes from 'prop-types';
-import { withTheme } from 'emotion-theming';
+import { withTheme } from '@emotion/react';
 import { themePropType } from '@sumup/design-tokens';
 
-export function ComponentWithInlineStyles({ theme, label }) {
+function ComponentWithInlineStyles({ theme, label }) {
   return <div style={{ color: theme.colors.p500 }}>{label}</div>;
 }
 
 ComponentWithInlineStyles.propTypes = {
   theme: themePropType.isRequired,
   label: PropTypes.string,
 };
+
+export default function withTheme(ComponentWithInlineStyles);
```

Comment:
Changed this to make the snippet technically correct (the export should be wrapped in `withTheme`) but ideally people should not use the HOC (but the hook) and not use prop-types (but TypeScript) 😛 Maybe we can add TS examples to this in a follow-up PR (similar to what I wrote in the migration guide)